crates/iota-swarm-config/src/node_config_builder.rs

```diff
@@ -397,6 +401,16 @@ impl FullnodeConfigBuilder {
         self
     }
 
+    pub fn with_grpc_api_address(mut self, addr: SocketAddr) -> Self {
```

Comment:
Let's pass the full config instead of only the address.

Reply:
Agreed. Fixed it at [383315b](https://github.com/iotaledger/iota/pull/7511/commits/383315b5073aeb4892861df2153345c229cca96a).